src/Google/Auth/AppIdentity.php

```diff
@@ -1,118 +0,0 @@
-<?php
-/*
- * Copyright 2014 Google Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/*
- * WARNING - this class depends on the Google App Engine PHP library
- * which is 5.3 and above only, so if you include this in a PHP 5.2
- * setup or one without 5.3 things will blow up.
- */
-use google\appengine\api\app_identity\AppIdentityService;
-
-if (!class_exists('Google_Client')) {
-  require_once dirname(__FILE__) . '/../autoload.php';
-}
-
-/**
- * Authentication via the Google App Engine App Identity service.
- */
-class Google_Auth_AppIdentity implements Google_Auth_Interface
```

Comment:
My understanding is the functionality of this class will be added to [`google/auth`](https://github.com/google/google-auth-library-php)